### PR TITLE
Update menu

### PR DESCRIPTION
**Pull request by wonmaungthein**
Created places, home and questionnaire route inside the admin page. eg.  url/admin/places ,   url/admin/home and url/admin/questionnaire.

### CODE REVIEW

**Comment by wonmaungthein** [comment]
[ADMIN / Update Admin Menu](https://trello.com/c/WMveOUHA/42-admin-update-admin-menu)

File: client/package.json [hunk 1]
```diff
@@ -18,5 +18,7 @@
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
   },
-  "devDependencies": {}
+  "devDependencies": {
+        "axios-mock-adapter": "^1.9.0"
+  }
```
Comment (rarmatei) [inline client/package.json]: We are not using this. Can we please remove?

File: server/routes/admin.js [hunk 2]
```diff
@@ -7,13 +7,21 @@ router.get('/', function (req, res, next) {
...
+router.get('/home', function (req, res, next) {
+  res.render('admin/home/home', {title: 'Home', name: 'Admin' });
 });
 
+router.get('/questionnaire', function (req, res, next) {
+  res.render('admin/questionnaire/questionnaire', {title: 'Questionnaire', name: 'Admin' });
+});
 
+router.get('/places/add', function (req, res, next) {
+  res.render('admin/places/placeAdd', { maintitle: 'Admin', title: 'Admin', name: 'KPC' });
```
Comment (rarmatei) [inline server/routes/admin.js]: Could we please remove `maintitle` property from here and from the page? 

We already have a `title` property.

File: server/views/admin/home/home.hbs [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1>{{title}}</h1>
+<p>Welcome to the {{title}} page of {{name}}, These are the places from {{name}}/{{title}}</p>
```
Comment (rarmatei) [inline server/views/admin/home/home.hbs]: Even though it's just an example placeholder, this text only makes sense for the Places page. Could you please change it on this one and on questionnaire.hbs?

Comment (wonmaungthein) [reply server/views/admin/home/home.hbs]: Thank you so much Rarmatei. I will fix them all..

File: server/routes/admin.js (hunk 2 above)
Comment (wonmaungthein) [reply server/routes/admin.js]: Sure, I have fixed this, thank you..

File: client/package.json (hunk 1 above)
Comment (wonmaungthein) [reply client/package.json]: Rarmatei, I think we are using "axios", right? Not so sure.

Comment (rarmatei) [reply client/package.json]: You're right, we are using Axios, but not the "mock-adapter", which is what you have added here

Comment (wonmaungthein) [reply client/package.json]: Oh, I see, I will delete that. Thank you..

**Pull request closed without merging**